.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,142 @@
+name: Build and Test
+
+on:
+  push:
+    branches: [ main ]
+    paths-ignore: [ '*.md' ]
+  pull_request:
+    branches: [ '**' ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - name: Install black and link shellcheck into expected location
+        run: |
```

Comment:
What is this python step doing?

Reply:
Installing `black` enables the use of gradle `spotless` to reformat python the same way it does for Java code.

The other line installs the `shellcheck` executable which is used by the `shellcheck` gradle plugin to run shellcheck on bash script files.

Both of these are nice to keep python and bash code relatively clean and consistent, but are only needed if you plan to have python and bash code in the repo.

Reply:
If you were to use them, you'd have to also set up the gradle plugins, for an example see: https://github.com/DataBiosphere/terra-data-catalog/blob/main/scripts/build.gradle

Reply:
Thanks for the context, @pshapiro4broad . Since we don't yet have any bash or python code in the repo yet, I've removed this step.